<launch>
  <!-- 启动人物追踪+F阵型节点 -->
  <node pkg="liu_formation_track" type="robot_person_following_node" name="robot_person_following_node" output="screen">
    <!-- 配置参数，可在此调整 -->
    <param name="topic_person"  value="/pose_person_following"/>
    <param name="topic_robot"   value="/pose_particle_localizer"/>
    <param name="topic_object"  value="/closest_objectpoint"/>

    <!-- 若人物话题已经是 map 坐标，设 true，否则 false -->
    <param name="person_pose_is_global" value="false"/>

    <!-- 超时/阵型参数 -->
    <param name="person_timeout"  value="3.0"/>
    <param name="enter_dist_max"  value="2.0"/>
    <param name="enter_fov_deg"   value="45.0"/>
    <param name="hold_dist_max"   value="2.5"/>
    <param name="hold_fov_deg"    value="60.0"/>
    <param name="max_v"           value="0.3"/>
    <param name="min_stop_radius" value="0.7"/>
    <param name="dead_zone"       value="0.3"/>
  </node>
</launch>
